docs/current/quickstart/snippets/publish/index.mjs

```diff
@@ -0,0 +1,42 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  // use a node:16 container
+  // mount the source code directory on the host
+  // at /src in the container
+  const source = await client.container()
+    .from("node:16")
+    .withMountedDirectory('/src', client.host().directory('.', { exclude: ["node_modules/", "ci/"] } ))
+
+  // set the working directory in the container
+  // install application dependencies
+  const runner = source
+    .withWorkdir("/src")
+    .withExec(["npm", "install"])
+
+   // run application tests
+  await runner
+    .withExec(["npm", "test", "--", "--watchAll=false"])
+    .exitCode()
+
+  // build application
+  // write the build output to the host
+  await runner
+    .withExec(["npm", "run", "build"])
+    .directory("./build")
+    .export("./build")
+
+  // highlight-start
+  // use an nginx:alpine container
+  // copy the build/ directory into the container filesystem
+  // at the nginx server root
+  // publish the resulting container to a registry
+  const imageRef = await client.container()
+    .from("nginx:alpine")
+    .withDirectory('/usr/share/nginx/html', client.host().directory('./build'))
```

Comment:
maybe refactor the `clien.host` call to a different variable so code is more readable?